app/views.py

```diff
@@ -2,9 +2,17 @@
 Routing file that holds the information for the separate webpage links.
 /index: main page for the spellchecker
 """
+
 from app import app
 from flask import render_template, request
 from .spell_check import *
+from .context import *
+
+def _fix_encoding(string):
+    return string.encode('iso-8859-1').decode('utf8')
+
+def _fix_encoding(string):
```

Comment:
Did you mean to define this function twice? They look the same to me.

Reply:
nope, dont quite know where the second came from